test/controllers/IncomeControllerSpec.scala

```diff
@@ -313,6 +312,9 @@ class IncomeControllerSpec extends PlaySpec
         when(taxAccountService.updateEstimatedIncome(any(), any(), any(), any())(any())).
```

Comment:
Unrelated to this PR but the toggle on/off expectations are the same?

Reply:
Slightly different in the views that are resolved